vision/tools/blob_annotator/main.py

```diff
@@ -22,6 +22,7 @@ class PascalVocAnnotator(object):
     CONFIG_PATH = 'config.json'
     SUPPORTED_FILE_EXTENSIONS = ('.jpg', '.png')
 
+
     def __init__(self, path_to_image_folder=TEST_DIRECTORY):
         ## Read config
```

Comment:
Functions are lacking docstrings throughout these files.

Reply:
I agree. All of the classes and files have docstrings though and I think that is sufficient for now.